### PR TITLE
Adds build and tests for OpenDataHub Intel Machine Learning Jupyter Notebooks Container

### PR DESCRIPTION
**Pull request by sharvil10**

### CODE REVIEW

File: ci-operator/config/opendatahub-io/notebooks/opendatahub-io-notebooks-main.yaml [hunk 1]
```diff
@@ -2,7 +2,7 @@ build_root:
   image_stream_tag:
     name: release
     namespace: openshift
-    tag: rhel-8-release-golang-1.19-openshift-4.14
+    tag: rhel-8-release-golang-1.17-openshift-4.10
```
Comment (harshad16) [inline ci-operator/config/opendatahub-io/notebooks/opendatahub-io-notebooks-main.yaml:5]: can you please revert this change?

File: ci-operator/config/opendatahub-io/notebooks/opendatahub-io-notebooks-main.yaml [hunk 2]
```diff
@@ -149,13 +157,6 @@ images:
   dockerfile_path: Dockerfile
   from: cuda-c9s-python-3.9
   to: cuda-rstudio-c9s-python-3.9
-- context_dir: base/anaconda-python-3.8
-  dockerfile_path: Dockerfile
-  to: base-anaconda-python-3.8
-- context_dir: jupyter/datascience/anaconda-python-3.8
-  dockerfile_path: Dockerfile
-  from: base-anaconda-python-3.8
-  to: jupyter-datascience-anaconda-python-3.8
```
Comment (harshad16) [inline ci-operator/config/opendatahub-io/notebooks/opendatahub-io-notebooks-main.yaml:158]: can you please revert this change?

File: ci-operator/config/opendatahub-io/notebooks/opendatahub-io-notebooks-main.yaml [hunk 3]
```diff
@@ -871,16 +910,6 @@ tests:
...
-- as: jupyter-datascience-anaconda-python-3-8-pr-image-mirror
-  run_if_changed: (base\/anaconda-python-3.8\/*)|(jupyter\/datascience\/anaconda-python-3.8\/*)
-  steps:
-    dependencies:
-      SOURCE_IMAGE_REF: jupyter-datascience-anaconda-python-3.8
-    env:
-      IMAGE_REPO: workbench-images
-      IMAGE_TAG: YearIndex
-      RELEASE_VERSION: jupyter-datascience-anaconda-python-3.8
-    workflow: opendatahub-io-ci-image-mirror
```
Comment (harshad16) [inline ci-operator/config/opendatahub-io/notebooks/opendatahub-io-notebooks-main.yaml:883]: can you please revert this change?

**Review by harshad16 — CHANGES_REQUESTED** [review]
Hello @sharvil10 
Thank you for the PR
can you address these changes

**Comment by harshad16** [comment]
@sharvil10 please make these changes.

File: ci-operator/config/opendatahub-io/notebooks/opendatahub-io-notebooks-main.yaml (hunk 1 above)
Comment (sharvil10) [reply ci-operator/config/opendatahub-io/notebooks/opendatahub-io-notebooks-main.yaml:5]: Updated

File: ci-operator/config/opendatahub-io/notebooks/opendatahub-io-notebooks-main.yaml (hunk 2 above)
Comment (sharvil10) [reply ci-operator/config/opendatahub-io/notebooks/opendatahub-io-notebooks-main.yaml:158]: Updated

File: ci-operator/config/opendatahub-io/notebooks/opendatahub-io-notebooks-main.yaml (hunk 3 above)
Comment (sharvil10) [reply ci-operator/config/opendatahub-io/notebooks/opendatahub-io-notebooks-main.yaml:883]: Updated

**Comment by harshad16** [comment]
@sharvil10 , i dont think you pushed your changes
there are no new changes, please take a look.

**Comment by sharvil10** [comment]
> @sharvil10 , i dont think you pushed your changes there are no new changes, please take a look.

https://github.com/openshift/release/pull/49889/commits/722510660fb0cc8d04172ff108bf7daa14a96751 is the commit with the change.

**Review by harshad16 — COMMENTED** [review]
/lgtm
/approve
:+1:

**Pull request merged**